server/package-lock.json

```diff
@@ -1,10532 +1,8 @@
 {
   "name": "health-career-access-program-server",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
lockfile version should be 2. Not sure why husky or package json allowed you to do this. :thinking:

Reply:
I ran all the command .... husky is pre-commit hook right ...?